srcpkgs/python3-mpmath/template

```diff
@@ -1,17 +1,22 @@
 # Template file for 'python3-mpmath'
 pkgname=python3-mpmath
-version=1.2.1
-revision=4
+version=1.3.0
+revision=1
 build_style=python3-module
-hostmakedepends="python3-setuptools_scm"
+hostmakedepends="python3-setuptools"
 depends="python3 python3-gmpy2"
-checkdepends="python3-pytest"
+checkdepends="python3-pytest python3-gmpy2"
```

Comment:
Could just add `${depends}` to avoid duplication if more depends are added in the future.

Reply:
Done